packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -233,6 +233,11 @@ const elements = [
 	'li',
 	'figure',
 	'nav',
+	'header',
+	'section',
+	'aside',
+	'footer',
+	'main',
```

Comment:
`form` is also missing on this list. See: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html

Reply:
Not sure what you would do with `form` since there are generally no granular blocks to make sense of it. Blocks that perform form-like functions would already define the element for themselves.